### DISCUSSION TITLE
In the world of classical music, which . . .

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#80] [Galaxy Qs] [answered]
In the world of classical music, which Austrian composer, renowned for his symphonies, is sometimes referred to as the "Father of the Symphony" and is famous for his "Surprise Symphony"?

**Comment by pvhuwung** [comment, marked as answer]
In the world of classical music, the Austrian composer renowned for his symphonies and sometimes referred to as the "Father of the Symphony" is ***Franz Joseph Haydn***. He is famous for his "Surprise Symphony" (Symphony No. 94 in G major), known for its unexpected loud chord that interrupts a soft, lyrical passage, surprising the audience and giving the symphony its name. 🎼🇦🇹
